specification/healthdataaiservices/resource-manager/Microsoft.HealthDataAIServices/readme.md

```diff
@@ -0,0 +1,78 @@
+# healthdataaiservices
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for healthdataaiservices.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the healthdataaiservices.
+
+```yaml
+openapi-type: arm
+openapi-subtype: rpaas
+tag: package-2024-02-28-preview
+```
+
+### Tag: package-2024-02-28-preview
+
+These settings apply only when `--tag=package-2024-02-28-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-02-28-preview'
+input-file:
+  - Microsoft.HealthDataAIServices/preview/2024-02-28-preview/openapi.json
```

Comment:
```suggestion
  - preview/2024-02-28-preview/openapi.json
```

Path is relative to `readme.md`, which is already in folder `Microsoft.HealthDataAIServices`.